build/mediaelement-and-player.js

```diff
@@ -2407,6 +2406,81 @@ window.MediaElement = mejs.MediaElement;
 
 }(mejs.i18n.locale.strings));
 /*!
+ * This is a i18n.locale language object.
```

Comment:
Yeah, I think what @astephenb said is correct. I had missed the Gruntfile changes in #23, so I probably missed some of the build artifacts between #22 and #23. 😥